ImageGallery/src/org/sleuthkit/autopsy/imagegallery/datamodel/DrawableDB.java

```diff
@@ -391,14 +391,6 @@ private boolean initializeDB() {
             return false;
         }
 
-        // @@@ BC: I don't think this is needed because it is the primary key. 
-        try (Statement stmt = con.createStatement()) {
```

Comment:
My impression is that it gets maintained as an additional index and we are therefore wasting time on inserts  and space.